python/lsst/pipe/tasks/dcrAssembleCoadd.py

```diff
@@ -168,7 +166,18 @@ def setDefaults(self):
         # The deepCoadd and nImage files will be overwritten by this Task, so don't write them the first time
         self.assembleStaticSkyModel.doNImage = False
         self.assembleStaticSkyModel.doWrite = False
-        self.detection.detection.thresholdValue = 10.0
+        self.detectSourcePsf.returnOriginalFootprints = False
+        self.detectSourcePsf.thresholdPolarity = 'positive'
+        # Only use bright sources for PSF measurement
+        self.detectSourcePsf.thresholdValue = 50
+        self.detectSourcePsf.nSigmaToGrow = 2
+        # A valid star for PSF measurement should at least fill 5x5 pixels
+        self.detectSourcePsf.minPixels = 25
+        # Use the variance plane to calculate signal to noise
+        self.detectSourcePsf.thresholdType = "pixel_stdev"
+        # The signal to noise limit is good enough, while the flux limit is set
+        # in dimensionless units and may not be appropriate for all data sets.
+        self.measurePsf.starSelector['objectSize'].doFluxLimit = False
```

Comment:
Where do the majority of these configs come from? 💯 for comments! I just want to check that it makes sense to set these defaults here and not somewhere where they would apply more widely.

Reply:
We definitely don't want these settings for the detection Task in general. I could imagine creating thin wrappers around the base detection Task with sensible names and default settings geared towards different use cases, and putting these settings in one geared towards detecting sources for PSF measurement. That sounds like an RFC, though...